### PR TITLE
Fix loading schema2 payloads for non-RPC requests

### PR DESCRIPTION
**Pull request by rsdy**
I think the decoding here is in the wrong place. The filename should be encoded, while the payload is raw.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "jest": "26.0.1",
     "jest-cli": "26.0.1",
     "jest-junit": "10.0.0",
-    "nanomsg": "4.1.0",
+    "nanomsg": "^4.1.0",
```
Comment (deepakprabhakara) [inline package.json]: let's pin versions

Comment (karlnorling) [reply package.json:26]: Let's go with `4.2.0`

File: root/usr/bin/redsift/protocol.js [hunk 2]
```diff
@@ -19,22 +14,31 @@ function flattenNestedArrays(value) {
   return [value];
 }
 
+function getFileContent(d) {
+  return fs.readFileSync(
```
Comment (karlnorling) [inline root/usr/bin/redsift/protocol.js:18]: Ideally we can speed up the read here by doing an array of `readFile` and resolve their promises. But I see that as a future improvement.

**Review by karlnorling — APPROVED** [review]
✅  We should just pin the version in package.json of `nanomsg`

**Review by deepakprabhakara — APPROVED** [review]

**Pull request merged**